qa/SidechainTestFramework/scutil.py

```diff
@@ -1089,4 +1090,52 @@ def get_account_balance(sc_node, address):
     return sc_node.wallet_getBalance(
         json.dumps({"address": str(address)}))["result"]["balance"]
 
+def computeForgedTxFee(sc_node, tx_hash, tracing_on=False):
+    transactionJson = sc_node.rpc_eth_getTransactionByHash(tx_hash)['result']
```

Comment:
Actually that line does not throw an error, but `transactionJson` will be `None` , and the condition is checked in the next line.

Reply:
That is true if the transaction was not found but the call itself was valid and executed successfully. If for some reason the call itself fails, e.g. because of "invalid params", "internal error" or any bug on the RPC handler we get an error response like
```
{
  "jsonrpc" : "2.0",
  "id" : "1",
  "error" : {
    "code" : -32602,
    "message" : "Invalid params"
  }
}
```
In that case `response["result"]` will throw and we don't even see the actual error but the generic python error for accessing a non-existent key.

Reply:
Ok you are right, I added error handling